jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/codescan/CodeWhispererCodeScanManager.kt

```diff
@@ -142,7 +144,7 @@ internal class CodeWhispererCodeScanManager(val project: Project) {
             afterCodeScan()
             val duration = (Instant.now().toEpochMilli() - startTime).toDouble()
             CodeWhispererTelemetryService.getInstance().sendSecurityScanEvent(
-                CodeScanTelemetryEvent(codeScanResponseContext, duration, codeScanStatus)
+                CodeScanTelemetryEvent(codeScanResponseContext, duration, codeScanStatus, projectSize.toDouble())
```

Comment:
nit: why define projectSize as Double instead of Long

Reply:
Because the projectSize is defined as `double` in the [telemetry](https://github.com/aws/aws-toolkit-common/blob/master/telemetry/definitions/commonDefinitions.json#L407). It should be `Long`, IMHO.

Reply:
The main reason why I define it as `double` is just because there was no usage of `long` in the https://github.com/aws/aws-toolkit-common/blob/master/telemetry/definitions/commonDefinitions.json for all fields, not for technical reason; I agree `long` is better here, can you make the update then for both packages ?